taglib/riff/wav/wavfile.cpp

```diff
@@ -47,7 +48,8 @@ class RIFF::WAV::File::FilePrivate
     properties(0),
     tagChunkID("ID3 "),
     hasID3v2(false),
-    hasInfo(false)
+    hasInfo(false),
+    acidInfo(NULL)
```

Comment:
I wonder if ```NULL``` is compatible with all compilers. I think it's better to initialize it with zero like other fields.